nixos/tests/packagekit.nix

```diff
@@ -13,12 +13,12 @@ import ./make-test.nix ({ pkgs, ... }: {
   };
 
   testScript = ''
-    startAll;
+    start_all()
 
     # send a dbus message to activate the service
-    $machine->succeed("dbus-send --system --type=method_call --print-reply --dest=org.freedesktop.PackageKit /org/freedesktop/PackageKit org.freedesktop.DBus.Introspectable.Introspect");
+    machine.succeed("dbus-send --system --type=method_call --print-reply --dest=org.freedesktop.PackageKit /org/freedesktop/PackageKit org.freedesktop.DBus.Introspectable.Introspect")
 
     # so now it should be running
-    $machine->succeed("systemctl is-active packagekit.service");
+    machine.succeed("systemctl is-active packagekit.service")
```

Comment:
Same for systemd

Reply:
I believe this should be `wait_for_unit`.